### PR TITLE
Adding Brazil

### PR DESCRIPTION
**Pull request by jbredeche**
depends on https://github.com/quantopian/trading_calendars/pull/32 and a new release of `trading_calendars`

### CODE REVIEW

File: tests/pipeline/test_domain.py [hunk 1]
```diff
@@ -333,6 +334,7 @@ def test_built_in_equity_calendar_domain_defaults(self):
             AT_EQUITIES: datetime.time(8, 15),
             AU_EQUITIES: datetime.time(9, 15),
             BE_EQUITIES: datetime.time(8, 15),
+            BR_EQUITIES: datetime.time(9, 15).
```
Comment (dmichalowicz) [inline tests/pipeline/test_domain.py]: Typo, period meant to be a comma.

File: zipline/pipeline/domain.py [hunk 2]
```diff
@@ -187,6 +187,7 @@ def __repr__(self):
 AT_EQUITIES = EquityCalendarDomain(CountryCode.AUSTRIA, 'XWBO')
 AU_EQUITIES = EquityCalendarDomain(CountryCode.AUSTRALIA, 'XASX')
 BE_EQUITIES = EquityCalendarDomain(CountryCode.BELGIUM, 'XBRU')
+BR_EQUITIES = EquityCalendarDomain(CountryCode.BRAZIL, 'BVMF')
```
Comment (dmichalowicz) [inline zipline/pipeline/domain.py]: You'll also want to add this to `BUILT_IN_DOMAINS` below.

**Comment by jbredeche** [comment]
thanks @dmichalowicz , updated

File: etc/requirements.txt [hunk 3]
```diff
@@ -82,7 +82,7 @@ empyrical==0.5.0
 tables==3.4.3
 
 # For trading calendars
-trading-calendars==1.4.0
+trading-calendars==1.4.1
```
Comment (dmichalowicz) [inline etc/requirements.txt]: Once we cut another micro release/update this, you'll also need to update `conda/trading-calendars/meta.yaml` like seen in https://github.com/quantopian/zipline/pull/2341/files

**Pull request merged**